compose-tests/src/androidTest/java/radiography/test/compose/ComposeUiTest.kt

```diff
@@ -595,9 +612,9 @@ class ComposeUiTest {
 
     assertThat(hierarchy).isEqualTo(
       """
-      |CompositionLocalProvider:
-      |${BLANK}CompositionLocalProvider { test-tag:"parent" }
-      |${BLANK}╰─LazyColumn { test-tag:"list", vertical-scroll-axis-range:"ScrollAxisRange(value=0.0, maxValue=0.0)" }
+      |Box:
+      |${BLANK}Box { test-tag:"parent" }
+      |${BLANK}╰─LazyColumn { vertical-scroll-axis-range:"ScrollAxisRange(value=0.0, maxValue=0.0)", test-tag:"list" }
```

Comment:
the order of the attributes changed, so I updated the expected value to match. Perhaps the renderer should sort them by name to keep them consistent?

Reply:
That seems like a good idea, but we can fix in a follow-up.